scripts/test.sh

```diff
@@ -0,0 +1,6 @@
+# Probably a better way to do this, but doesn't work if done like above.
```

Comment:
This shouldn't be included, it was just a work in progress isolation of the problem code below that still doesn't work

Reply:
removed.